rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -1295,6 +1295,358 @@ void testTxIsProcessableInLegacyVersion() {
         assertFalse(BridgeUtils.txIsProcessableInLegacyVersion(TxSenderAddressType.UNKNOWN, activations));
     }
 
+    @Test
+    void testIsMigrationTx_sending_funds_from_retired_p2sh_fed_to_active_p2sh_fed() {
+        NetworkParameters networkParameters = bridgeConstantsMainnet.getBtcParams();
+        Context btcContext = new Context(networkParameters);
+        when(activations.isActive(ConsensusRule.RSKIP201)).thenReturn(true);
+
+        List<BtcECKey> retiredFederationKeys = Stream.of(
+            BtcECKey.fromPrivate(Hex.decode("fc01")),
+            BtcECKey.fromPrivate(Hex.decode("fc02"))
+        ).sorted(BtcECKey.PUBKEY_COMPARATOR).collect(Collectors.toList());
+
+        Federation retiredFederation = new P2shErpFederation(
+            FederationTestUtils.getFederationMembersWithBtcKeys(retiredFederationKeys),
+            Instant.ofEpochMilli(1000L),
+            1L,
+            bridgeConstantsMainnet.getBtcParams(),
+            bridgeConstantsMainnet.getErpFedPubKeysList(),
+            bridgeConstantsMainnet.getErpFedActivationDelay(),
+            activations
+        );
+
+        List<BtcECKey> activeFederationKeys = Stream.of(
+            BtcECKey.fromPrivate(Hex.decode("fa01")),
+            BtcECKey.fromPrivate(Hex.decode("fa02"))
```

Comment:
The same here, an actual private key-like hex would be better. Or is this some kind of a seed?

Reply:
Same answered.